include/spdlog/details/spdlog_impl.h

```diff
@@ -31,6 +31,7 @@
 #include "../sinks/file_sinks.h"
 #include "../sinks/stdout_sinks.h"
 #include "../sinks/syslog_sink.h"
+#include "../sinks/android_sink.h"
```

Comment:
No need to include this file. It is not used from splog_imple.h as far as i can tell